paasta_tools/utils.py

```diff
@@ -2995,6 +2996,27 @@ def get_hostname() -> str:
     return socket.getfqdn()
 
 
+def get_files_of_type_in_dir(
+    service: str = None, soa_dir: str = DEFAULT_SOA_DIR, file_type: str = None
+) -> List[str]:
+    """Recursively search path if type of file exists.
+
+    :param file_name: a string of a type of a file (kubernetes, slo, etc.)
+    :param path: a string path
+    :return: a list
+    """
+    service = "**" if service is None else service
+    soa_dir = DEFAULT_SOA_DIR if soa_dir is None else soa_dir
+    file_type += "-*.yaml"
```

Comment:
should we make `file_type` required? right now this will error out if no `file_type` is provided upon trying to add a string to None

Reply:
Ooh yes.